core/proto/accountmanager/v1/accountmanager.proto

```diff
@@ -0,0 +1,156 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Can we follow the same naming scheme in `core/proto`, i.e., no `v1` directory?

If we want to introduce a `v1` directory, we can do that in a separate PR. We can have `core/proto/v1` and all protos goes there.

Reply:
I am doing what I can to follow the directory structure outlined in the [style guide](https://cloud.google.com/apis/design/directory_structure). We previously didn't think we could follow the style guide in the open source, but I think I've been able to solve that in this PR.  If this pattern is successful, I will move the other protos to follow suit.